docs/_data/sidebar.yml

```diff
@@ -145,6 +145,9 @@
     ja: SOQL
   baseUrl: /soql/
   children:
+     - title:
+        en: SOQL Builder
```

Comment:
We need to add the japanese entry too or the site breaks

Reply:
@lcampos - added it back. :)